frontend/app/src/App.vue

```diff
@@ -177,7 +178,7 @@ export default class App extends Vue {
     });
 
     if (process.env.NODE_ENV === 'development' && this.logged) {
-      monitor.start();
+      monitor.start(this.periodicClientQueryPeriod);
```

Comment:
You could also access the store directly in the `monitoring.ts` but you could also just pass a default value in the start method and call it without the period here. Since this is development code it doesn't really matter. this is only here to restart the monitoring when the dev server does hot reloading.

Reply:
I went the way of using store in `monitoring.ts` file, I find it cleaner overall as `monitor.start()` is used in several places.